### ISSUE TITLE
Documentation reports incorrect version

### ISSUE THREAD

**Issue by Quafadas** [scalacenter/scalajs-bundler#415]
I think that the front of this page should be; 
https://scalacenter.github.io/scalajs-bundler/

addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.20.0")

rather than 

addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.0.0+1-b3702414-SNAPSHOT")

**Comment by ptrdom** [comment]
Looking at CI setup, it should fix itself up upon next commit to main, but we should probably only publish to GitHub Pages only on tags, rather than every commit to main - https://github.com/scalacenter/scalajs-bundler/blob/main/.github/workflows/ghpages.yml#L7.

**Comment by OndrejSpanel** [comment]
I agree. The documentation should reference a stable version, not a SNAPSHOT one. At the moment this is perhaps:

```
addSbtPlugin("ch.epfl.scala" % "sbt-scalajs-bundler" % "0.21.0")
```

The #421 claims to fix this, but still there is a snapshot version listed at https://scalacenter.github.io/scalajs-bundler/
